build-tools/build-image/cross/Dockerfile

```diff
@@ -21,12 +21,14 @@ ENV GOARM 6
 ENV KUBE_DYNAMIC_CROSSPLATFORMS \
   armel \
   arm64 \
+  s390x \
   ppc64el
 
 ENV KUBE_CROSSPLATFORMS \
   linux/386 \
   linux/arm linux/arm64 \
   linux/ppc64le \
+  linux/s390x \
```

Comment:
we probably need to split this PR into two parts, since we need a new cross-build (and other) images before the cross-build will work in Jenkins.